openmetadata-spec/src/main/resources/json/schema/entity/applications/configuration/external/collateAIAppConfig.json

```diff
@@ -1,26 +1,27 @@
 {
-  "$id": "https://open-metadata.org/schema/entity/applications/configuration/external/metaPilotAppConfig.json",
+  "$id": "https://open-metadata.org/schema/entity/applications/configuration/external/collateAIAppConfig.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "MetaPilotAppConfig",
-  "description": "Configuration for the MetaPilot External Application.",
+  "title": "CollateAIAppConfig",
+  "description": "Configuration for the CollateAI External Application.",
   "type": "object",
-  "javaType": "org.openmetadata.schema.entity.app.external.MetaPilotAppConfig",
+  "javaType": "org.openmetadata.schema.entity.app.external.CollateAIAppConfig",
   "definitions": {
-    "metaPilotAppType": {
+    "collateAIAppType": {
       "description": "Application type.",
       "type": "string",
-      "enum": ["MetaPilot"],
-      "default": "MetaPilot"
+      "enum": ["CollateAI"],
```

Comment:
Would we need to replace this in the code anywhere?

Reply:
the code lives in Collate, so updating it there. We still need to put collate-only apps here since we don't know how to extend the `oneOf` dynamically